packages/@aws-cdk/aws-appmesh/lib/shared-interfaces.ts

```diff
@@ -173,11 +173,11 @@ class FileAccessLog extends AccessLog {
  */
 export interface BackendDefaults {
   /**
-   * Client policy for backend defaults
+   * TLS properties for Client policy for backend defaults
    *
    * @default - none
    */
-  readonly tlsClientPolicy?: TlsClientPolicy;
+  readonly clientPolicyTls?: ClientPolicyTlsOptions;
```

Comment:
I think it is ok to leave as it is.

I agree "correct" way might be what you have suggested (since `ClientPolicy` really represents a policy that belongs to the client); however, all docs like AWS user guide and CloudFormation refer `ClientPolicy` as one entity and I think it is easier for reader to be consistent.

In addition, `clientTlsPolicy` could give an impression that it represents `Policy` attribute for the TLS configuration (considering we have `TlsValidation` and `TlsCertificate`).

Reply:
Then let's call it `clientPolicy`.

`ClientPolicyTls` just doesn't make sense.

Reply:
By looking at the CloudFormation doc, we actually have 

- [`ClientPolicy`](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-appmesh-virtualnode-clientpolicy.html) - object that represents client policy
- [`ClientPolicyTls`](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-appmesh-virtualnode-clientpolicytls.html) - object that represents TLS property for a client policy.

I think calling this as a `ClientPolicy` is misleading because this is presenting TLS property.

Reply:
Can we then leave it as it is today?

I think `ClientPolicyTlsOptions` is a strictly worse name than `TlsClientPolicy`.

Reply:
Got it. WIll revert the change.